### PR TITLE
Ajoute la page de profil et portfolio pour les utilisateurs et freela…

### PR DESCRIPTION
**Pull request by Aymane-Ennahli**
Acceder a la route http://localhost:3000/fr/dashboard pour le test

Modifications : 
utilisation de template nextAdmin (https://nextjstemplates.com/templates/nextadmin)Docs :(https://docs.nextadmin.co/)

- Modification de tailwind.config.ts
- Ajout des nouveaux components 
-  Ajout de repertoire : src/css
-  Ajout de repertoire : src/fonts
- Ajout de repertoire : src/js
- Ajout de gook : src/hooks/useLocalStorage.tsx
- Ajout de hook : src/hooks/useColorMode.tsx
- Modication de : src/app/[locale]/dashboard/@freelancer/page.tsx
- Ajout de fichier : src/app/[locale]/dashboard/@freelancer/forms/form-elements/page.tsx
- Ajout de fichier : src/app/[locale]/dashboard/@freelancer/pages/settings/page.tsx

### CODE REVIEW

File: src/app/[locale]/dashboard/@freelancer/forms/form-elements/page.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import FormElements from "@/components/FormElements";
+import { Metadata } from "next";
+import DefaultLayout from "@/components/Layouts/DefaultLaout";
+
```
Comment (codeonym) [inline src/app/[locale]/dashboard/@freelancer/forms/form-elements/page.tsx:5]: Remove metadata file -- i has already been edited on root layout

File: src/app/[locale]/dashboard/@freelancer/page.tsx [hunk 2]
```diff
@@ -1,11 +1,23 @@
-import React from 'react';
+import Breadcrumb from "@/components/Breadcrumbs/Breadcrumb";
+import { Metadata } from "next";
+import DefaultLayout from "@/components/Layouts/DefaultLaout";
+import ProfileBox from "@/components/ProfileBox";
```
Comment (codeonym) [inline src/app/[locale]/dashboard/@freelancer/page.tsx:5]: remove meta data from here

File: src/app/[locale]/dashboard/@freelancer/pages/settings/page.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import Breadcrumb from "@/components/Breadcrumbs/Breadcrumb";
+
+import { Metadata } from "next";
+import DefaultLayout from "@/components/Layouts/DefaultLaout";
+import SettingBoxes from "@/components/SettingBoxes";
+
```
Comment (codeonym) [inline src/app/[locale]/dashboard/@freelancer/pages/settings/page.tsx:6]: remove metadata from here

File: src/components/Breadcrumbs/Breadcrumb.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import Link from "next/link";
+
+interface BreadcrumbProps {
+  pageName: string;
+}
+
+const Breadcrumb = ({ pageName }: BreadcrumbProps) => {
+  return (
```
Comment (codeonym) [inline src/components/Breadcrumbs/Breadcrumb.tsx:8]: Create a segmented list of urls based on pathname ( seperation via '/')  
Notice: urls must be routable ( valid url within _Link_ component)

File: src/components/CalenderBox/index.tsx [hunk 5]
```diff
@@ -0,0 +1,237 @@
+const CalendarBox = () => {
```
Comment (codeonym) [inline src/components/CalenderBox/index.tsx:1]: Canlendar is hardcoded, better be removed

File: src/js/us-aea-en.js
Comment (codeonym) [inline src/js/us-aea-en.js:1]: Whats this file about?

File: tailwind.config.ts
Comment (codeonym) [inline tailwind.config.ts:1]: dont chaange use as is , if there is a need to define a custom style add it to extention

File: src/components/Layouts/DefaultLaout.tsx [hunk 6]
```diff
@@ -0,0 +1,39 @@
+"use client";
```
Comment (codeonym) [inline src/components/Layouts/DefaultLaout.tsx:1]: dont use use-client in layout, and refactor the layout code to the nextjs special file: layout.tsx

**Review by codeonym — CHANGES_REQUESTED** [review]
Address the following issues:

1. Dont use use client when not needed
2. Remove metadata from pages

File: src/app/[locale]/dashboard/@freelancer/forms/form-elements/page.tsx (hunk 1 above)
Comment (Aymane-Ennahli) [reply src/app/[locale]/dashboard/@freelancer/forms/form-elements/page.tsx:5]: Done

**Comment by codeonym** [comment]
Pull request cancelled due to code change

**Pull request closed without merging**